packages/block-editor/src/components/link-control/link-preview.js

```diff
@@ -56,8 +61,22 @@ export default function LinkPreview( {
 
 	let icon;
 
+	const attachmentImg =
+		value.type === 'attachment' && isImageUrl( value?.url )
+			? value.url
+			: null;
+
 	if ( richData?.icon ) {
 		icon = <img src={ richData?.icon } alt="" />;
+	} else if ( attachmentImg ) {
+		//TODO we don't have an alt text for this image
```

Comment:
An empty `alt` tag is acceptable when there is no way to provide an accurate description